provisioners.proto

```diff
@@ -139,9 +139,17 @@ message ACMEProvisioner {
 		TLS_ALPN_O1 = 3;
 		DEVICE_ATTEST_01 = 4;
 	}
+	enum PlatformType {
+		NONE = 0;
```

Comment:
Would this be a meaningful "platform", or is `Unknown` better here? If the use case is that none of the other types are used, then `None` seems OK.

Reply:
`UNKNOWN` cannot be used because it collides with ChallengeType's UNKNOWN.

Reply:
Bummer. Well, let's go with this then for now. We'll probably have to iterate on it a bit anyway

Reply:
I won't tag it for now. Just use @main to use it.